src/cljs/harja/views/hallinta/kojelauta.cljs

```diff
@@ -74,13 +75,47 @@
       (r/wrap (:urakat valinnat) #(e! (tiedot/->AsetaSuodatin :urakat %)))]]]])
 
 
+(defn valikatselmus-sarake
+  [rivi]
+  (let [{:keys [rahapaatokset lupauspaatokset hoitokauden_alkuvuosi]} rivi
+        edellisen-hoitokauden-alkuvuosi (- (pvm/vuosi (first (pvm/paivamaaran-hoitokausi (pvm/nyt)))) 1)
+        ;; 15.11. on takaraja, milloin edellisen hoitokauden välikatselmus pitää olla tehtynä (edellinen --> kuluva hk -1)
+        valikatselmuksen-takaraja-ohi? (or
+                                         (< hoitokauden_alkuvuosi edellisen-hoitokauden-alkuvuosi)
+                                         (and
+                                           (= hoitokauden_alkuvuosi edellisen-hoitokauden-alkuvuosi)
+                                           (> (pvm/nyt)
+                                             (kustannusten-seuranta-tiedot/valikatselmuksen-takarajapvm (+ hoitokauden_alkuvuosi 1)))))]
+    [yleiset/wrap-if true
+     [yleiset/tooltip {} :% "Siirry kustannusten seurantaan"]
+     [:a.klikattava {:on-click #(siirtymat/kustannusten-seurantaan-valitussa-urakassa (:ely_id rivi) (:id rivi))}
+      [:div.rahapaatokset
+       (if (nil? rahapaatokset)
+         (yleiset/tila-indikaattori (if valikatselmuksen-takaraja-ohi? "hylatty" "kesken")
+           {:fmt-fn (constantly "Ei katto- tai tavoitehintapäätöksiä")})
+         (for [rp rahapaatokset]
+           ^{:key (hash rp)}
+           [:span
+            (yleiset/tila-indikaattori "valmis"
+              {:fmt-fn (constantly (kustannusten-seuranta-tiedot/valikatselmuksen-paatostyypin-nimi rp))})]))]
+      [:div.lupauspaatokset
+
+       (if (nil? lupauspaatokset)
+         (yleiset/tila-indikaattori (if valikatselmuksen-takaraja-ohi? "hylatty" "kesken")
+           {:fmt-fn (constantly "Ei lupauspäätöksiä")})
+         (for [lp lupauspaatokset]
+           ^{:key (hash lp)}
+           [:span
+            (yleiset/tila-indikaattori "valmis"
+              {:fmt-fn (constantly (kustannusten-seuranta-tiedot/valikatselmuksen-paatostyypin-nimi lp))})]))]]]))
+
 (defn kustannussuunitelman-tila-sarake
   [rivi]
   (let [indeksi-saatavilla? (boolean (:indeksikerroin rivi))
         {:keys [aloittamattomia vahvistamattomia vahvistettuja suunnitelman_tila]} (:ks_tila rivi)]
     [yleiset/wrap-if true
      [yleiset/tooltip {} :% "Siirry kustannussuunnitelmaan"]
```

Comment:
Kustiksen sijaan, me voitaisiin siirtyä suoraan välikatselmukseen. 

Tämä kuitenkin vaati hieman virvelöintiä, koska meidän täytyy ensin odottaa että valittu urakka on ladannut ennenkuin voidaan siirtyä välikatselmukseen, jotenka, tässä on toimiva koodi jonka voit halutessas lisätä:

<details>
<summary> ᴄʟɪᴄᴋ ᴛᴏ ᴇxᴘᴀɴᴅ </summary>

```clojure 

[yleiset/wrap-if true
     [yleiset/tooltip {} :% "Siirry välikatselmukseen"]
     [:a.klikattava {:on-click #(do
	  ;; Valitse urakka
	  (siirtymat/kustannusten-seurantaan-valitussa-urakassa (:ely_id rivi) (:id rivi))
	  ;; Pollaa kunnes valittu urakka id on klikattu urakka ^ 
	  (let [max-yritykset 20
			yritykset (atom 0)
			interval-id (atom nil)]

		(reset! interval-id
		  (js/setInterval
			(fn []
			  (println "\n __poll...")

			  ;; Nosta yritysmäärää
			  (swap! yritykset inc)

			  ;; Odota, että valittu urakka on ladannut 
			  (if (= (:id @nav/valittu-urakka) (:id rivi))
				(do
				  ;; Urakka on ladannut -> clearInterval -> avaa-valikatselmus
				  (println "__call: löytyi id: " (:id rivi) "  valittuun urakkaan: " (:id @nav/valittu-urakka))
				  (js/clearInterval @interval-id)
				  (.setTimeout js/window
							   (fn []
								 (siirtymat/avaa-valikatselmus
								   ;; Palauttaa halutun välikatselmuksen aikavälin, 
								   ;; esim: [#object[Object 20221001T000000] #object[Object 20230930T235959]]
								   (nth (urakka/hoito-tai-sopimuskaudet @nav/valittu-urakka)
									 (dec (urakka/hoitokauden-jarjestysnumero hoitokauden_alkuvuosi
											(-> @tila/yleiset :urakka :loppupvm))))))
							   1800))
				;; Lopeta pollaus jos tuloksia ei ikinä tule 
				(when (>= @yritykset max-yritykset)
				  (println "__call: max yritysmäärä saavutettiin -> clearInterval")
				  (js/clearInterval @interval-id))))
			550))) ;; Pollaa joka 0.5s
	  )}

```

</details>

⚠️ HOX : 
Tässä on Tonilla tällä hetkellä työn alla bugikorjaus, jossa **hoitokauden-jarjestysnumero** ei toimi oikein urakoille joilla 6 hoitokautta (tällä hetkellä ongelma on pelkästään toi imatran urakka id: 458), tähän varmaan tulossa sieltäpäin korjaus. Kaikille muille urakoille ton pitäis toimia.